AirSkyBoat/AirSkyBoat
# Change Xolotl and firends from superlink to sublink

**_I affirm:_**


## Please enter a player-facing description

Xolotl and their adds will no longer link with Antlions.

## What does this pull request do? (Please be technical)

Changes Xolotl and their adds to use mobmod sublink instead of mobmod superlink.
This stops Xolotl and the 2 adds from being added to every mob party in the zone.

For those unfamiliar, mobs of a similar type are actually grouped into parties by `CZoneEntities::FindPartyForMob(CBaseEntity* PEntity)`.  Superlink causes an entity to always pass the family/super family checks.

## Steps to test these changes

Xolotl does spawn his pets when idle.  I dont know if that is acruate or not.  Lets jump through some hoops anyways.
1. Aggro xol and let him spawn both pets.
1. !zone _notAttowha_ to clear all hate
1. Make sure GM mode is active - dont want to accidentally aggro
1. !gotoid 16806215
1. pull one of the skeles - all 3 mobs link
1. !zone and come back
1. pull the other skele
1. !zone and come back
1. pull Xol - this is a trick.  Xol has scripting to share hate so this already works
1. !speed 255 and go run north past the tracker antlions
1. Note that when xol and company pass the antlions - the antlion does not care.

## Special Deployment Considerations

Yes
- the mobmod superlink (26) must be removed from these mobs.

Old values
```
-- Xolotl
INSERT INTO `mob_spawn_mods` VALUES (16806215,26,32,1); -- SUPERLINK: 32

-- Xolotl'S Hound Warrior
INSERT INTO `mob_spawn_mods` VALUES (16806216,26,32,1); -- SUPERLINK: 32

-- Xolotl'S Sacrifice
INSERT INTO `mob_spawn_mods` VALUES (16806217,26,32,1); -- SUPERLINK: 32
```

Manual edits should be fine or the follwoing query will work
Verification Query
```
SELECT * FROM mob_spawn_mods WHERE mob_spawn_mods.mobid IN (16806215, 16806216, 16806217) AND mob_spawn_mods.modid = 26
```
Execution Query
```
DELETE FROM mob_spawn_mods WHERE mob_spawn_mods.mobid IN (16806215, 16806216, 16806217) AND mob_spawn_mods.modid = 26
```
